src/shared/lib/pre-handlers/gaugingstations.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const { errorHandler } = require('./lib/error-handler');
+const GaugingStationsService = require('shared/lib/connectors/services/water/GaugingStationsService');
+/**
+ * Loads a gauging station from the water service using the Gauging Station Id
+ * which can be passed in or retrieved from the request object
+ * @param {Object} request
+ * @param {String} companyId guid, if provided
+ */
+const loadGaugingStations = async (request, h, gaugingStationId = null) => {
```

Comment:
If this is a pre-handler, how would the third argument here be provided?  The signature is usually just `(request, h)`